src/Microsoft.Tye.Core/DockerContainerBuilder.cs

```diff
@@ -44,15 +44,15 @@ public static async Task BuildContainerImageAsync(OutputContext output, Applicat
             var dockerFilePath = Path.Combine(application.GetProjectDirectory(project), Path.GetDirectoryName(project.RelativeFilePath)!, "Dockerfile");
             if (File.Exists(dockerFilePath))
             {
-                output.WriteDebugLine($"Using existing dockerfile '{dockerFilePath}'.");
+                output.WriteDebugLine($"Using existing Dockerfile '{dockerFilePath}'.");
```

Comment:
Casing in `Dockerfile` is often important, due to it being a default file with the docker CLI, and since docker is used in case-sensitive file systems regularly - I figured being consistent inside and out of the code is important.